launchpad-stack-aws3/infra/launchpad.tf

```diff
@@ -126,11 +126,31 @@ resource "launchpad_config" "cluster" {
   spec {
     mcr {
       version = var.launchpad.mcr_version
+      repo_url = "https://repos.mirantis.com"
+      channel  = "stable"
     }
     mke {
       version        = var.launchpad.mke_version
+      image_repo     = "docker.io/Mirantis"
+      admin_username = var.launchpad.mke_connect.username
       admin_password = var.launchpad.mke_connect.password
-      install_flags  = []
+
+      install_flags = [
+        "--san=${local.MKE_URL}",
+      ]
+    }
+
+    dynamic "msr" {
+      for_each = local.has_msr ? [1] : []
+
+      content {
+        version    = var.launchpad.msr_version
+        image_repo = "docker.io/Mirantis"
```

Comment:
Repo name should be in lowercase (mirantis, instead of Mirantis); If not, launchpad throws an error.

Reply:
fixed